packages/core/strapi/lib/commands/__tests__/export.test.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+
+const utils = require('../transfer/utils');
+
+const mockDataTransfer = {
+  createLocalFileDestinationProvider: jest.fn(),
+  createLocalStrapiSourceProvider: jest.fn(() => {}),
```

Comment:
Just wondering if there is a reason why the source provider mock returns an object & the destination provider mock returns `undefined`?

Reply:
Good catch, probably I was just trying stuff to make something work and forgot to remove it. Thanks!